R/generateSyntheticData.R

```diff
@@ -55,7 +77,13 @@ generateSyntheticData <- function(dataset, n.vars, samples.per.cond, n.diffexp,
                                   fraction.non.overdispersed = 0, random.outlier.high.prob = 0, 
                                   random.outlier.low.prob = 0, single.outlier.high.prob = 0, 
                                   single.outlier.low.prob = 0, effect.size = 1.5, 
-                                  output.file = NULL) {
+                                  output.file = NULL,
+                                  tree = NULL, prop.var.tree = 1.0,
```

Comment:
All new arguments are optional. If not specified, the original behavior of the function should be preserved.